Client/Frontend/Settings/Main/Account/SyncNowSetting.swift

```diff
@@ -145,26 +145,29 @@ class SyncNowSetting: WithAccountSetting {
         // swiftlint:enable unused_setter_value
     }
 
-    private lazy var troubleshootButton: UIButton = {
-        let troubleshootButton = UIButton(type: .roundedRect)
+    private lazy var troubleshootButton: UIButton = .build { [weak self] troubleshootButton in
+        guard let self = self else { return }
         troubleshootButton.setTitle(.FirefoxSyncTroubleshootTitle, for: .normal)
         troubleshootButton.addTarget(self, action: #selector(self.troubleshoot), for: .touchUpInside)
-        troubleshootButton.tintColor = theme.colors.actionPrimary
-        troubleshootButton.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: 12, weight: .regular)
-        troubleshootButton.sizeToFit()
-        return troubleshootButton
-    }()
+        troubleshootButton.setTitleColor(self.theme.colors.actionPrimary, for: .normal)
+        troubleshootButton.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                                                     size: 12,
+                                                                                     weight: .regular)
+    }
+
+    private lazy var accessoryViewContainer: UIStackView = .build { stackView in
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 4
+    }
 
     private lazy var warningIcon: UIImageView = {
-        let imageView = UIImageView(image: UIImage(named: "AmberCaution"))
-        imageView.sizeToFit()
-        return imageView
+        return UIImageView(image: UIImage(named: "AmberCaution"))
```

Comment:
Should these names be in `ImageIdentifiers`?

Reply:
Yes probably 👀 Will do that